assets/js/plugins-screen.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Extra zazz for the WP Admin Plugins page.
+ * @see Admin_Plugins_Screen::enqueue_scripts_and_styles().
+ */
+( function( $ ){
```

Comment:
Was wondering if there's a way to inject these classes programmatically via PHP. I spent a few minutes investigating all the hooks in https://developer.wordpress.org/reference/classes/wp_plugins_list_table/, and it looks like the answer is no, but thought I'd ask just in case you know something magical? If there were a way, we could lose this .js file entirely.

Reply:
There isn't. I would also love to not do it this way, but customization of the plugins screen is quite limited.